test-registry/function_configs_yaml/echo.yaml

```diff
@@ -0,0 +1,16 @@
+name: echo
```

Comment:
In one organization, the whole function (with all files related) are in a single .tar.gz: https://github.com/open-lambda/open-lambda/blob/main/src/worker/lambda/handlerPuller.go#L110.  So let's not have a config in a different directory.  It should be in the same directory where the function code is.

Let's use the same config name for every lambda.  Maybe ol.yaml?  This would be similar to how we check to see if there is a requirements.txt file: https://github.com/open-lambda/open-lambda/blob/main/src/worker/lambda/lambdaFunction.go#L77.